client/components/info-popover/index.jsx

```diff
@@ -11,84 +11,82 @@ import Popover from 'components/popover';
 import classNames from 'classnames';
 import analytics from 'lib/analytics';
 
-export default React.createClass( {
+export default class InfoPopover extends Component {
+	static propTypes = {
+		autoRtl: PropTypes.bool,
+		className: PropTypes.string,
+		gaEventCategory: PropTypes.string,
+		id: PropTypes.string,
+		ignoreContext: PropTypes.shape( {
+			getDOMNode: PropTypes.func
+		} ),
+		popoverName: PropTypes.string,
+		position: PropTypes.oneOf( [
```

Comment:
That's exactly the kind of thing I was trying to catch with the stricter propType. The positioning behaviour of that popover differs from other popovers in [some](https://github.com/Automattic/wp-calypso/pull/13600#issuecomment-298934782) [unspecified](https://github.com/Automattic/wp-calypso/blob/9e631355f68b8d6ac864b6bf3f12d6bb61c071fe/client/components/popover/util.js#L279-L284) [way](https://github.com/Automattic/wp-calypso/blob/9e631355f68b8d6ac864b6bf3f12d6bb61c071fe/client/components/popover/util.js#L258-L263) and doesn't adhere to [the docs](https://github.com/Automattic/wp-calypso/blob/9e631355f68b8d6ac864b6bf3f12d6bb61c071fe/client/components/popover/README.md#position--string----default-top).

I've created #14403 to correct this case. After a quick grep of the codebase, it seems you came across the _only_ place this occurrs 🙂